### PR TITLE
Update startup to use latest package

### PR DESCRIPTION
**Pull request by virkt25**
- All imports now work using npm releases `@loopback/*`
- Fixed startup across microservices to not use Server as it's been removed from the latest alpha. 
- connect to #67

### CODE REVIEW

File: services/account/index.ts [hunk 1]
```diff
@@ -1,34 +1,41 @@
...
-    const uptime = Date.now() - this._startTime.getTime();
-    return { uptime: uptime };
-  }
-}
+const app = new Application();
+app.controller(AccountController);
+
+app.bind('servers.http.enabled').to(true);
+app.bind('servers.https.enabled').to(true);
+app.bind('applications.accounts').to(this);
```
Comment (bajtos) [inline services/account/index.ts]: I think these three lines are not needed at all.

File: services/account/index.ts [hunk 2]
```diff
@@ -1,34 +1,41 @@
-import { Application, Server } from 'loopback-next/packages/core';
+import { Application } from 'loopback-next/packages/core';
 import { AccountController } from './controllers/AccountController';
+import * as http from 'http';
```
Comment (bajtos) [inline services/account/index.ts]: Unused import?

File: services/facade/index.ts [hunk 3]
```diff
@@ -1,36 +1,9 @@
...
+const app = new Application();
+app.controller(AccountController);
 
-  constructor() {
-    super();
-    this.controller(AccountController);
-    this.bind('servers.http.enabled').to(true);
-    this.bind('servers.https.enabled').to(true);
-  }
+app.start();
```
Comment (b-admike) [inline services/facade/index.ts]: Can we catch errors if the app fails to start as we did before?

File: services/account-without-juggler/index.ts [hunk 4]
```diff
@@ -1,39 +1,13 @@
-import { Application, Server } from 'loopback-next/packages/core';
+import { Application } from '@loopback/core';
 import { AccountController } from './controllers/AccountController';
 import { AccountRepository } from './repositories/account';
 
-class AccountMicroservice extends Application {
```
Comment (bajtos) [inline services/account-without-juggler/index.ts]: I think it was intentional to have `AccountMicroservice extends Application`.  Is there any particular reason why you are preferring to use vanilla `Application` instance instead?

File: services/account-without-juggler/index.ts [hunk 5]
```diff
@@ -1,39 +1,13 @@
...
-    this._startTime = new Date();
-    const server = new Server(this, { port: 3001 });
-    server.bind('applications.accounts').to(this);
-    return server.start();
-  }
+app.start();
 
-  info() {
-    const uptime = Date.now() - this._startTime.getTime();
-    return { uptime: uptime };
```
Comment (bajtos) [inline services/account-without-juggler/index.ts]: Should we preserve `uptime` data?

File: services/account-without-juggler/index.ts [hunk 6]
```diff
@@ -1,39 +1,13 @@
...
+app.bind('http.port').to(3001);
+app.bind('repositories.account').toClass(AccountRepository);
 
-  async start() {
-    this._startTime = new Date();
-    const server = new Server(this, { port: 3001 });
-    server.bind('applications.accounts').to(this);
-    return server.start();
-  }
+app.start();
```
Comment (bajtos) [inline services/account-without-juggler/index.ts]: Please handle start errors.

File: services/account-without-juggler/index.ts [hunk 7]
```diff
@@ -1,39 +1,13 @@
...
-  const app = new AccountMicroservice();
-  await app.start();
-  console.log('Application Info:', app.info());
-}
-
-main().catch(err => {
-  console.log('Cannot start the app.', err);
-  process.exit(1);
-});
+console.log('Application started on port:', app.getSync('http.port'));
```
Comment (bajtos) [inline services/account-without-juggler/index.ts]: This message should wait until the promise returned by `app.start()` is resolved. When `http.port` is set to `0`, the value is updated to the real port when after the app starts listening. If your console log does not wait for `listen` to finish, then you may print incorrect port number.

Same comment applies to other app files changed by this pull request.

File: services/facade/index.ts (hunk 3 above)
Comment (bajtos) [inline services/facade/index.ts]: +1

**Comment by bajtos** [comment]
@virkt25 could you please update installation instructions in README? I think we no longer need to `git clone` loopback-next repo and `npm link` it, do we?

**Comment by virkt25** [comment]
@bajtos Feedback Applied. Restored Info & extending Application. Also updated the README & the `build` script.

**Pull request merged**